{
  "stats": {
    "suites": 4,
    "tests": 6,
    "hooks": 9,
    "passes": 2,
    "pending": 1,
    "failures": 2,
    "start": "2018-03-19T22:50:13.370Z",
    "end": "2018-03-19T22:50:18.540Z",
    "duration": 5170
  },
  "suites": [
    {
      "title": "test suite",
      "tests": [
        {
          "title": "test",
          "result": "passed",
          "err": {}
        },
        {
          "title": "skipped test",
          "result": "pending",
          "err": {}
        },
        {
          "title": "failing test",
          "result": "failed",
          "err": {
            "message": "false == true",
            "stack": "AssertionError [ERR_ASSERTION]: false == true\n    at Context.it (P:\\nodejs\\wdio-json-output-reporter\\test\\sample-test.js:7:35)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:590:26\n    at new Promise (<anonymous>)\n    at new F (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\core-js\\library\\modules\\_export.js:35:28)\n    at Context.executeSync (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:588:12)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:759:42",
            "type": "AssertionError [ERR_ASSERTION]",
            "expected": true,
            "actual": false
          }
        }
      ],
      "suites": [
        {
          "title": "nested suite",
          "tests": [
            {
              "title": "nested test",
              "result": "passed",
              "err": {}
            },
            {
              "title": "nested failing test",
              "result": "failed",
              "err": {
                "message": "false == true",
                "stack": "AssertionError [ERR_ASSERTION]: false == true\n    at Context.it (P:\\nodejs\\wdio-json-output-reporter\\test\\sample-test.js:11:44)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:590:26\n    at new Promise (<anonymous>)\n    at new F (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\core-js\\library\\modules\\_export.js:35:28)\n    at Context.executeSync (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:588:12)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:759:42",
                "type": "AssertionError [ERR_ASSERTION]",
                "expected": true,
                "actual": false
              }
            }
          ],
          "suites": []
        }
      ]
    }
  ],
  "pending": [
    {
      "title": "skipped test",
      "result": "pending",
      "err": {}
    }
  ],
  "failures": [
    {
      "title": "failing test",
      "result": "failed",
      "err": {
        "message": "false == true",
        "stack": "AssertionError [ERR_ASSERTION]: false == true\n    at Context.it (P:\\nodejs\\wdio-json-output-reporter\\test\\sample-test.js:7:35)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:590:26\n    at new Promise (<anonymous>)\n    at new F (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\core-js\\library\\modules\\_export.js:35:28)\n    at Context.executeSync (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:588:12)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:759:42",
        "type": "AssertionError [ERR_ASSERTION]",
        "expected": true,
        "actual": false
      }
    },
    {
      "title": "nested failing test",
      "result": "failed",
      "err": {
        "message": "false == true",
        "stack": "AssertionError [ERR_ASSERTION]: false == true\n    at Context.it (P:\\nodejs\\wdio-json-output-reporter\\test\\sample-test.js:11:44)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:590:26\n    at new Promise (<anonymous>)\n    at new F (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\core-js\\library\\modules\\_export.js:35:28)\n    at Context.executeSync (P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:588:12)\n    at P:\\nodejs\\wdio-json-output-reporter\\node_modules\\wdio-sync\\build\\index.js:759:42",
        "type": "AssertionError [ERR_ASSERTION]",
        "expected": true,
        "actual": false
      }
    }
  ],
  "passes": [
    {
      "title": "test",
      "result": "passed",
      "err": {}
    },
    {
      "title": "nested test",
      "result": "passed",
      "err": {}
    }
  ]
}